src/corelibs/U2Gui/src/options_panel/PrimerLineEdit.h

```diff
@@ -25,9 +25,11 @@
 #include <QLineEdit>
 #include <QValidator>
 
+#include <U2Core/global.h>
+
 namespace U2 {
 
-class PrimerLineEdit : public QLineEdit {
+class U2GUI_EXPORT PrimerLineEdit : public QLineEdit {
```

Comment:
You make this class a part of the public API. Could you please add some docs how it works?

Reply:
OK